elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -92,3 +96,13 @@ No symbolic expressions (those that involve fractions, square roots, variables,
 {{#shortformat}}
 {{#relabs}}(rtol={{rtol}}, atol={{atol}}){{/relabs}}{{#sigfig}}({{digits}} significant figures){{/sigfig}}{{#decdig}}({{digits}} digits after decimal){{/decdig}}
 {{/shortformat}}
+
+{{#format_error}}
+    <p> Invalid format: one or more entries of the submitted answer could not be interpreted as a decimal {{#complex}}or complex{{/complex}} number.  No symbolic input or expressions are allowed.</p>
+    <p> Example <i>valid</i> inputs: <code class="user-output">5</code> <code class="user-output">-17</code> <code class="user-output">4.25e7</code> {{#complex}}<code class="user-output">3.6i</code>{{/complex}} </p>
+    <p> Example <i>invalid</i> inputs: <code class="user-output-invalid">6+9</code> <code class="user-output-invalid">4.25*10^7</code> <code class="user-output-invalid">3*pi</code> {{^complex}}<code class="user-output-invalid">3.6i</code>{{/complex}} </p>
```

Comment:
I'm a bit worried about the fact that the invalid examples are the most noticeable things in the dialog, because they are red. Maybe we just shouldn't have invalid examples? Or maybe have them be the same color as the valid ones? I'm not sure.